### ISSUE TITLE
[BUG/ISSUE] Floating point exception error in AIRQNT in patch/13.2.1 branch

### ISSUE THREAD

**Issue by msulprizio** [geoschem/geos-chem#886]
GEOS-Chem is stalling in what seems to be a continuous loop when running with the current `patch/13.2.1` branch. When the code is built with `-DCMAKE_BUILD_MODE=Debug` and gfortran 10.2, it crashes in AIRQNT with the following error:

```
Program received signal SIGFPE: Floating-point exception - erroneous arithmetic operation.

Backtrace for this error:
#0  0x2aff7183f3ff in ???
#0  0x2aff7183f3ff in ???
#0  0x2aff7183f3ff in ???
#0  0x2aff7183f3ff in ???
#0  0x2aff7183f3ff in ???
#1  0x905d4a in __calc_met_mod_MOD_airqnt._omp_fn.1
        at /n/home05/msulprizio/RD/DiffTest/Test/CodeDir/src/GEOS-Chem/GeosCore/calc_met_mod.F90:526
#2  0x2aff70f620a1 in GOMP_parallel
        at ../.././libgomp/parallel.c:171
#3  0x8fc228 in __calc_met_mod_MOD_airqnt
        at /n/home05/msulprizio/RD/DiffTest/Test/CodeDir/src/GEOS-Chem/GeosCore/calc_met_mod.F90:324
#2  0x2aff70f699e5 in gomp_thread_start
        at ../.././libgomp/team.c:123
#3  0x2aff715f4ea4 in ???
#4  0x2aff719079fc in ???
#5  0xffffffffffffffff in ???
Floating point exception (core dumped)
```

I was able to narrow down the cause of this bug to commit 7f6021ca6e846771437b0e6ba1db078ff908dcdf.

**Comment by msulprizio** [comment]
The changes in commit 7f6021ca6e846771437b0e6ba1db078ff908dcdf seem to have introduced unintended changes in the grid definition. What should be:

```
Grid box latitude centers [degrees]: 
 -89.000  -86.000  -82.000  -78.000  -74.000  -70.000  -66.000  -62.000
 -58.000  -54.000  -50.000  -46.000  -42.000  -38.000  -34.000  -30.000
 -26.000  -22.000  -18.000  -14.000  -10.000   -6.000   -2.000    2.000
   6.000   10.000   14.000   18.000   22.000   26.000   30.000   34.000
  38.000   42.000   46.000   50.000   54.000   58.000   62.000   66.000
  70.000   74.000   78.000   82.000   86.000   89.000

Grid box latitude edges [degrees]: 
 -90.000  -88.000  -84.000  -80.000  -76.000  -72.000  -68.000  -64.000
 -60.000  -56.000  -52.000  -48.000  -44.000  -40.000  -36.000  -32.000
 -28.000  -24.000  -20.000  -16.000  -12.000   -8.000   -4.000    0.000
   4.000    8.000   12.000   16.000   20.000   24.000   28.000   32.000
  36.000   40.000   44.000   48.000   52.000   56.000   60.000   64.000
  68.000   72.000   76.000   80.000   84.000   88.000   90.000
```

is now:

```
Grid box latitude centers [degrees]: 
 -90.00000  -86.00000  -82.00000  -78.00000  -74.00000  -70.00000  -66.00000
 -62.00000  -58.00000  -54.00000  -50.00000  -46.00000  -42.00000  -38.00000
 -34.00000  -30.00000  -26.00000  -22.00000  -18.00000  -14.00000  -10.00000
  -6.00000   -2.00000    2.00000    6.00000   10.00000   14.00000   18.00000
  22.00000   26.00000   30.00000   34.00000   38.00000   42.00000   46.00000
  50.00000   54.00000   58.00000   62.00000   66.00000   70.00000   74.00000
  78.00000   82.00000   86.00000   89.00000

Grid box latitude edges [degrees]: 
 -92.00000  -88.00000  -84.00000  -80.00000  -76.00000  -72.00000  -68.00000
 -64.00000  -60.00000  -56.00000  -52.00000  -48.00000  -44.00000  -40.00000
 -36.00000  -32.00000  -28.00000  -24.00000  -20.00000  -16.00000  -12.00000
  -8.00000   -4.00000    0.00000    4.00000    8.00000   12.00000   16.00000
  20.00000   24.00000   28.00000   32.00000   36.00000   40.00000   44.00000
  48.00000   52.00000   56.00000   60.00000   64.00000   68.00000   72.00000
  76.00000   80.00000   84.00000   88.00000   90.00000
```

**Comment by msulprizio** [comment]
This is now fixed in commit https://github.com/geoschem/geos-chem/commit/1e6661b329c7717bce0eb8c2f108c47d94ae1655. The issue was caused by the removal of an ELSE statement when computing global latitude edges. This fix will be included in [13.2.1](https://github.com/geoschem/geos-chem/milestone/25).
